core/src/executor/select/error.rs

```diff
@@ -8,6 +8,9 @@ pub enum SelectError {
     #[error("table alias for blend not found: {0}")]
     BlendTableAliasNotFound(String),
 
+    #[error("column reference {0} is ambiguous, please specify the table name")]
```

Comment:
@panarch @ever0de 
I've added a new error for ambiguous column ref in `SelectError` part because it seems like to happen specifically when using `SELECT` and `JOIN`.
But I am not so sure about this. Should I move this to `TranslateError` or not?

Reply:
This looks obvious. We now have a new `validate` module in `plan`.
This means that the location we need to add an error is inside of `plan`.
How about adding `PlanError` to `plan/error.rs`'?

Reply:
Sounds great!